lib/PuppeteerSharp.Tests/Page/EvaluateOnNewDocumentTests.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace PuppeteerSharp.Tests.Page
+{
+    [Collection("PuppeteerLoaderFixture collection")]
+    public class EvaluateOnNewDocumentTests : PuppeteerPageBaseTest
+    {
+        [Fact]
+        public async Task ShouldEvaluateBeforeAnythingElseOnThePage()
+        {
+            await Page.EvaluateOnNewDocumentAsync(@"function(){
```

Comment:
Do you want to add docs to `EvaluateOnNewDocumentAsync`?

Reply:
Yup